utils/skopeo.py

```diff
@@ -0,0 +1,239 @@
+import json
+import logging
+import re
+import subprocess
+
+from distutils import spawn
+
+
+class SkopeoCmdError(Exception):
+    """
+    Indicates that the skopeo command failed.
+    """
+
+
+class Skopeo:
+    """Wrapper around the skopeo utility."""
+
+    default_scheme = 'docker://'
+    default_repo = ''
+    default_tag = 'latest'
+
+    def __init__(self, dry_run=False):
+        self.dry_run = dry_run
+        self.log = logging.getLogger(__name__)
```

Comment:
why? we just use the global logger elsewhere in the project, should we change this?

Reply:
I'd to avoid that. Using the root logger, `%(name)s` will always be `root`, regerdless the module you log the message from.

Reply:
makes sense